src/handler.cpp

```diff
@@ -864,6 +865,11 @@ void affect_total(struct char_data * ch)
 
   apply_drug_modifiers_to_ch(ch);
 
+  // Part of the otaku simulation; modifies otaku attributes depending on drugs, cyber, etc.
+  if (ch->persona && ch->persona->type == ICON_LIVING_PERSONA) {
+    update_otaku_deck(ch, ch->persona->decker->deck);
```

Comment:
Please check for `persona->decker` and `persona->decker->deck`. An ugly error message is always better than a crash!